### ISSUE TITLE
0.17 beta does not install transpiler

### ISSUE THREAD

**Issue by teohhanhui** [jspm/jspm-cli#1458]
`jspm init` does not install the transpiler [`babel`]

I had to do `jspm install plugin-babel` manually.

**Comment by guybedford** [comment]
See https://github.com/jspm/jspm-cli/releases/tag/0.17.0-beta.3.

**Comment by teohhanhui** [comment]
@guybedford

> By default transpilers are plugins, with the init process providing the standard plugins for Traceur, Babel and TypeScript.

By "providing" I assumed it means "installing".

**Comment by teohhanhui** [comment]
The 0.17 beta guide shows this line in the output:

```
ok   Installed dev dependency plugin-babel to npm:systemjs-plugin-babel@^0.0.2 (0.0.2)
```

but it did not happen when I tried it.

**Comment by teohhanhui** [comment]
Looks like this has been fixed in 0.17.0-beta.5

Thank you.
